ffig/FFIG.py

```diff
@@ -78,14 +78,47 @@ def build_model_from_source(path_to_source, module_name):
     Returns:
     - model built from a clang.cindex TranslationUnit with a name from args
     """
-    tu = clang.cindex.TranslationUnit.from_source(
-        path_to_source, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    try:
+        tu = clang.cindex.TranslationUnit.from_source(
+            path_to_source, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    except TranslationUnitLoadError:
```

Comment:
I fear that libclang may be rather too good at coping with bad input. We could probably fail if there are any diagnostics at all. Better to be fussy than not.

```
class NoClosingSemicolon { }
```

should suffice for simple test input